collection-spec/collection-spec.md

```diff
@@ -90,10 +90,10 @@ and consumers have not been granted any explicit right to use the data.
 
 Collections are *strongly recommended* to provide summaries of the values of fields that they can expect from the `properties` 
 of STAC Items contained in this Collection. This enables users to get a good sense of what the ranges and potential values of 
-different fields in the Collection are, without to inspect a number of Items (or crawl them exhaustively to get a definitive answer). 
+different fields in the Collection are, without having to inspect a number of Items (or crawl them exhaustively to get a definitive answer). 
 Summaries help to fully define Collections, especially if they don't link to any Items. They also give clients enough information to 
 build tailored user interfaces for querying the data, by presenting the potential values that are available.
-Summaries should summarize all values in every Item underneath the collection, including in any nested sub-Catalogs.
+ Fields selected to be included in summaries should consider all the field's values in every Item underneath the collection, including in any nested sub-Catalogs.
```

Comment:
'consider' feels a bit weak to me, like they could think about using it and decide not to.

```suggestion
 Fields selected to be included in summaries should capture all the potential values of the field that appear in every Item underneath the collection, including in any nested sub-Catalogs.
```